doc/releases/release-notes-4.2.rst

```diff
@@ -111,6 +111,11 @@ Deprecated APIs and options
   was deprecated since Zephyr 4.0, and users were advised to migrate to alternative
   crypto backends.
 
+* The :kconfig:option:`CONFIG_BT_MESH_USES_TINYCRYPT` Kconfig option has been removed. It
+  was deprecated since Zephyr 4.0. Users were advised to use
+  :kconfig:option:`CONFIG_BT_MESH_USES_MBEDTLS_PSA` or
```

Comment:
Is it easy to find out which one should I use PSA or TFM_PSA, or is it platform dependent? Because this statement gives a choice to the readers, without explaining how to make that choice, neither giving any reference where to find more information to make this choice.

Reply:
This is the release notes. To figure out what exactly to use, please read documentation.